### ISSUE TITLE
make broken with latest VCG on Ubuntu

### ISSUE THREAD

**Issue by spikeheap** [cdcseacave/openMVS#176]
- Ubuntu 16.04
- Latest versions of everything else

### Steps to reproduce the problem

Follow the steps from the [wiki](https://github.com/cdcseacave/openMVS/wiki/Building) in a fresh install of Ubuntu 16.04. I've tested this using Docker (because I'm on Mac) using the `ubuntu:16.04` image. 

The `make -j2` command fails (build output below). 

This is tied to the version of VCG, so I can raise the issue there if it's more appropriate. The three most recent commits to the VCG repo are:
- [094789](https://github.com/cdcseacave/VCG/commit/0947893864950d863d00ea4eb126eebd898241de) fails (this is the latest tip of `master`).
- [e85984](https://github.com/cdcseacave/VCG/commit/e85984608b36433036c72e99034b4cecad62cae4) fails (expected, the subsequent commit is titled 'fix linux').
- [a306cd](https://github.com/cdcseacave/VCG/commit/a306cdf07738d735e86d9f0fd84d46459c1405c5) succeeds.

A workaround for now would be to update the wiki to reference the working commit after the clone of VCG:

```
cd vcglib
git checkout b42e3ed7fa1a3861fd35060d40e8cbfe58278100
cd ..
```

The build output is too long for this issue, so I've put it into a gist: https://gist.github.com/spikeheap/b3cb859c7fb7acd06399582013970120

**Comment by etiennekruger** [comment]
Just ran into the same issue on Debian Jessie and checking out the commit stated solves the problem.

**Comment by cdcseacave** [comment]
The fix to compile is easy, unfortunately the latest VCG does have a bug affecting the mesh decimation functionality (as well as the latest Meshlab). I am trying to solve this issue with the VCG developer.

**Comment by cdcseacave** [comment]
fixed in 2a41453
